modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBulkLoadSelfTest.java

```diff
@@ -154,21 +184,21 @@ private CacheConfiguration cacheConfigWithQueryEntity() {
      *
      * @return true if we are testing near cache.
      */
-    protected abstract boolean nearCache();
+    protected boolean nearCache(){ return isNear; };
```

Comment:
Can we also remove this methods and use variables?

Reply:
yep